Jenkinsfile.integration

```diff
@@ -66,13 +67,21 @@ pipeline {
                         if (sourceChanged[i].contains("tox.ini")) {
                             ToxChanged = true
                         }
+                        if (sourceChanged[i].contains("Jenkinsfile.integration")) {
+                            JenkinsfileChanged = true
```

Comment:
A cleaner way would be that if Jenkinsfile.integration has changed, then set ToxChanged/DocsChanged/PlaybooksChanged to true. It would not need the extra conditionals.